src/index.html

```diff
@@ -0,0 +1,133 @@
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- SECOND VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./secnd.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.web" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- THIRD VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./3rd.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- FORTH VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./coding.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+
+            <!-- FIFTH VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./hqdefault.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
```

Comment:
- [ ] You should consider giving a descriptive [`alt`](https://www.w3schools.com/tags/att_img_alt.asp) message.
A  descriptive **alt** attributes to image tags improves on the accessibility to blind individuals and also slow network. Kindly consider adding the `alt` attribute to `<img>` tag.